python/risk_meters/mttr_vulns.py

```diff
@@ -0,0 +1,176 @@
+# Get the historical vulnerability counts for each month for a year.
+# The vulnerability counts for 'high', 'medium', and 'low' risk scores and include
+# risk accepted vulnerability counts.
+#
+# This program is written to run stand-alone or as a subprocess invoked from list_risk_scores.py.
+
+import os
+import sys
+import calendar
+from datetime import date, timedelta
+import requests
+from prettytable import PrettyTable
+
+def sign_bit(num):
+    return (int(num < 0))
+
+def get_next_month(year, month):
+    month_day = calendar.monthrange(year, month)
+    num_days_per_month = month_day[1]
+    month_delta = timedelta(days=num_days_per_month)
+
+    month_first_day = date(year, month, 1)
+    next_month_first_day = month_first_day + month_delta
+
+    return (next_month_first_day.year, next_month_first_day.month)
+
+# Obtain risk meter information by searching all the risk meters.
+# There is no show asset group (risk meter) API.
```

Comment:
nitpick: there is no show asset group API which can list risk meters by name param, right?

Reply:
That's correct.  The "Show Asset Group" API uses the asset group ID.